sources/world/Makefile

```diff
@@ -123,24 +123,25 @@ clean: ## Remove output files
 
 
 # Data without metrics
-../../data/world_countries_v7.geo.json: ../../data/admin_regions_lvl2_v2.geo.json /tmp/additional-regions.geo.json ## World Countries GeoJSON full size
+../../data/world_countries_v7.geo.json: ../../data/admin_regions_lvl2_v2.geo.json /tmp/additional-regions.geo.json ./country-codes.csv ## World Countries GeoJSON full size
 	codesA=$$(csvsql --query 'select distinct in_region_iso from "duplicated-regions" where not keep' duplicated-regions.csv | tail +2 | sed -e 's/^/"/g' -e 's/$$/"/g' | paste -sd "," -) && codesB=$$(csvsql --query 'select distinct out_region_iso2 as iso from "duplicated-regions" where keep' duplicated-regions.csv | tail +2 | sed -e 's/^/"/g' -e 's/$$/"/g' | paste -sd "," -) && mapshaper -i $(DATA_DIR)/admin_regions_lvl2_v2.geo.json \
   -filter "[$${codesA},$${codesB}].indexOf(region_iso_code) == -1" \
   -dissolve 'country_iso2_code' 'copy-fields=country_iso3_code,country_name' \
   -rename-fields "iso2=country_iso2_code,iso3=country_iso3_code,name=country_name" \
+  -join ./country-codes.csv "keys=iso2,iso2" "fields=iso_numeric" \
```

Comment:
I think we need to add the `iso_numeric` codes as strings rather than integers. Maybe using the `string-fields` option in the [Mapshaper `-join`](https://github.com/mbloch/mapshaper/wiki/Command-Reference#-join)?